syntax = "proto3";

package fileio;

service FileIO {
  rpc RequestSqlite (SqliteRequest) returns (SqliteResponse) {}
}

message SqliteRequest {
  string filepath = 1;
}

message SqliteResponse {
  uint32 status = 1;   // 0 is success
  bytes sqlite = 2;
}
